backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Services/Implementation/PolicyAdministrationPoint.cs

```diff
@@ -230,17 +248,25 @@ private async Task<bool> WriteDelegationPolicyInternal(string policyPath, List<R
                         return false;
                     }
 
+                    ServiceResource resource = null;
+                    if (!string.IsNullOrWhiteSpace(resourceRegistryId))
```

Comment:
This has already been retrieved in the
if (resourceMatchType == ResourceAttributeMatchType.ResourceRegistry) scope (line 160)
and can be removed